app/pages/analyst/application/[applicationId]/assessments/screening.tsx

```diff
@@ -74,9 +77,10 @@ const ScreeningAssessment = ({
           schema={screening}
           uiSchema={screeningUiSchema}
           noValidate
-          formData={applicationByRowId.assessmentForm?.jsonData}
-          onChange={() => {
+          formData={formData}
+          onChange={(e) => {
             setIsFormSaved(false);
+            setFormData({ ...e.formData });
```

Comment:
Do we have to make this change in other places as well?

Reply:
No I don't believe so since the application form auto saves it doesn't have this issue though we will make sure the rest of the assessment pages use this as well. I think we can make a reusable assessments form when we go to do the rest of them.